cocos/scripting/js-bindings/manual/jsb_dragonbones_manual.cpp

```diff
@@ -66,6 +66,36 @@ static bool js_cocos2dx_dragonbones_Armature_getDisplay(se::State& s)
 }
 SE_BIND_FUNC(js_cocos2dx_dragonbones_Armature_getDisplay)
 
+static bool js_cocos2dx_dragonbones_Armature_getSlots(se::State& s)
+{
+	dragonBones::Armature* cobj = (dragonBones::Armature*)s.nativeThisObject();
+	const std::vector<dragonBones::Slot *, std::allocator<dragonBones::Slot *> >& result = cobj->getSlots();
+	se::HandleObject arr(se::Object::createArrayObject(result.size()));
+
+	uint32_t i = 0;
+	se::Value tmp;
+	bool ok = true;
+	for (const auto& slot : result)
+	{
+		if (!native_ptr_to_rooted_seval<dragonBones::Slot>(slot, &tmp))
+		{
+			ok = false;
+			break;
+		}
+		arr->setArrayElement(i, tmp);
+		++i;
+	}
+	if (ok)
+		s.rval().setObject(arr);
+	else
+		s.rval().setUndefined();
```

Comment:
else s.rval().setUndefined() isn't needed. If we don't set s.rval, it's `undefined` by default.